packages/shared/src/components/search/SearchProgressBar.module.css

```diff
@@ -0,0 +1,13 @@
+.animatedBar {
+  background-image: linear-gradient(to right, rgba(113, 71, 237, 0), rgb(206, 61, 243));
+  animation: fadeLeftToRight 1.2s linear infinite;
+}
+
+@keyframes fadeLeftToRight {
```

Comment:
Pulse didnt work?

https://tailwindcss.com/docs/animation#pulse

Reply:
Unfortunately, the pulse goes full width with the animation.